Python/Product/Uap/Interpreter/PythonUapInterpreter.cs

```diff
@@ -0,0 +1,237 @@
+﻿/* ****************************************************************************
+ *
+ * Copyright (c) Microsoft Corporation. 
+ *
+ * This source code is subject to terms and conditions of the Apache License, Version 2.0. A 
+ * copy of the license can be found in the License.html file at the root of this distribution. If 
+ * you cannot locate the Apache License, Version 2.0, please send an email to 
+ * vspython@microsoft.com. By using this source code in any fashion, you are agreeing to be bound 
+ * by the terms of the Apache License, Version 2.0.
+ *
+ * You must not remove this notice, or any other, from this software.
+ *
+ * ***************************************************************************/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PythonTools.Analysis;
+using Microsoft.PythonTools.Interpreter;
+using Microsoft.VisualStudioTools;
+
+namespace Microsoft.PythonTools.Uap.Interpreter {
+    class PythonUapInterpreter : IPythonInterpreter, IPythonInterpreterWithProjectReferences2, IDisposable {
```

Comment:
That is not exactly right in this case.  References to files end up getting included in the appx package, which is needed for extension modules that are not already included.  This also allows for extension module project references to be included and to be built along with the appx as well.  Even though we are not currently reflecting upon the reference for intellisense because of limitations, it is still a valuable way to include the additional extension modules.  We could just say that you need to just add the file as content, but references seemed to flow as a concept...especially for project references.